ignite/templates/module/create/files/base/x/{{moduleName}}/module/autocli.go.plush

```diff
@@ -0,0 +1,37 @@
+package <%= moduleName %>
+
+import (
+	autocliv1 "cosmossdk.io/api/cosmos/autocli/v1"
+
+	modulev1 "<%= modulePath %>/api/<%= appName %>/<%= moduleName %>"
+)
+
+// AutoCLIOptions implements the autocli.HasAutoCLIConfig interface.
+func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
+	return &autocliv1.ModuleOptions{
+		Query: &autocliv1.ServiceCommandDescriptor{
+			Service: modulev1.Query_ServiceDesc.ServiceName,
+			RpcCommandOptions: []*autocliv1.RpcCommandOptions{
+				{
+					RpcMethod: "Params",
+					Use:       "params",
+					Short:     "Shows the parameters of the module",
+				},
+
+				// this line is used by starport scaffolding # 1
```

Comment:
since we are changing the logic can we use more specific placeholder?
eg:

```suggestion
				// this line is used by starport scaffolding # stargate/module/autocli/params
```

Reply:
tbh I disagree as these will be the only two things getting scaffolded in that file and just for that we need to create a new placeholder everywhere.

Reply:
We never know if we need to change or increase the placeholders of this file in the future. It takes work to manage later. And we need to think about the placeholder that can always stay the same so we can keep the support.

wdyt @ilgooz @jeronimoalbi @Ehsan-saradar ?

Reply:
Alright, I'll update.